packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1463,6 +1469,17 @@ class _CoversSameAreaAs extends Matcher {
     description.add('covers expected area and only expected area');
 }
 
+Future<ui.Image> _toImage(Element element) {
```

Comment:
toFoo(bar) usually means "convert this object into foo using bar as the configuration", but here you're converting bar. Maybe call this something like "_captureImage"?

Reply:
done